### PR TITLE
feat: add annotation campaigns

### PR DESCRIPTION
**Pull request by raphael0202**
Implements https://github.com/openfoodfacts/robotoff/issues/947

### CODE REVIEW

File: robotoff/app/api.py [hunk 1]
```diff
@@ -1166,20 +1169,24 @@ def on_get(self, req: falcon.Request, resp: falcon.Response):
         response: JSONType = {}
         page: int = req.get_param_as_int("page", min_value=1, default=1)
         count: int = req.get_param_as_int("count", min_value=1, default=25)
-        question_type: str = req.get_param("type")
-        value_tag: str = req.get_param("value_tag")
+        insight_type: str = req.get_param("type")
```
[the remark this answers is not in the document]
Comment (alexgarel) [reply robotoff/app/api.py:1172]: hum, I was looking also for campaign addition but it seems this API is not yet documented ?

**Review by alexgarel — APPROVED** [review]
LGTM, just a remark on an API that is not yet documented ?

File: robotoff/app/api.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (raphael0202) [reply robotoff/app/api.py:1172]: No it has only been documented in the previous Blueprint documentation, it should definitely be added.
There is no endpoint yet for campaign addition

[the remark this answers is not in the document]
Comment (raphael0202) [reply robotoff/app/api.py:1172]: I've documented the route :+1:

**Pull request merged**